entrypoint.sh

```diff
@@ -25,6 +25,29 @@ for arg in "$@"; do
 	esac
 done
 
-# start, and pass all remaining flags to tracee-rules
+# create named pipe for communicating between tracee-ebpf and tracee-rules
+tracee_pipe=/tmp/tracee/pipe
+rm -f $tracee_pipe && mkfifo $tracee_pipe
+
+# start tracee-ebpf in the background
 EVENTS=$($TRACEE_RULES_EXE --list-events)
-$TRACEE_EBPF_EXE --output=format:gob --output=option:parse-arguments --trace event=$EVENTS,mem_prot_alert | $TRACEE_RULES_EXE --input-tracee=file:stdin --input-tracee=format:gob $@
+echo "starting tracee-ebpf..."
+$TRACEE_EBPF_EXE --output=format:gob --output=option:parse-arguments --trace event=$EVENTS --output=out-file:$tracee_pipe &
```

Comment:
missing mem_prot_alert event

Reply:
I removed it on purpose. it's included in the `$EVENTS`